modules/core/src/main/java/org/apache/ignite/internal/commandline/cache/argument/PartitionReconciliationCommandArg.java

```diff
@@ -36,8 +36,8 @@
     /** If {@code true} - print data to result with sensitive information: keys and values. */
     INCLUDE_SENSITIVE("--include-sensitive", Boolean.FALSE),
 
-    /** Percent of system loading between 0 and 1. */
-    LOAD_FACTOR("--load-factor", 1d),
+    /** Maximum number of threads that can be involved in reconciliation activities. */
+    PARALLELISM("--parallelism", 0),
```

Comment:
Don't forget to update control.sh_cache_help.output

Reply:
Fixed